### PR TITLE
Fix(#954) - Implement the new home page design

### LINKED ISSUE #954: Implement the new home page design
## Describe the solution you'd like

- Implement the new home-page design - https://xd.adobe.com/view/688fb08d-f097-464d-9c1a-8dd7cb6d312f-3ddc/screen/c5408c6e-6099-4924-87e1-5d086502e031/ 

## Describe alternatives you've considered

None

## Additional context

N/A

### PR DESCRIPTION
**Pull request by hybridx**
## Resolves #954 

# Explain the feature/fix

- Design implementation https://xd.adobe.com/view/688fb08d-f097-464d-9c1a-8dd7cb6d312f-3ddc/screen/c5408c6e-6099-4924-87e1-5d086502e031/  

## Does this PR introduce a breaking change

No

## Screenshots

Screenshot_2021-05-26 Home One Platform

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by deshmukhmayur — APPROVED** [review]

File: packages/home-spa/src/contact-us/contact-us.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    {
+      name: 'Sumeet Ingole',
+      title: 'Software Engineer',
+      gender: 'male'
+    },
+    {
+      name: 'Deepesh Nair',
+      title: 'Software Engineer',
+      gender: 'male'
+    },
```
Comment (riginoommen) [inline packages/home-spa/src/contact-us/contact-us.js:42]: Where is prasad?

Comment (hybridx) [reply packages/home-spa/src/contact-us/contact-us.js:42]: Alright I'll add him

File: packages/home-spa/src/service/service.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
-const listHomeType = `
-query {
-  listHomeType {
+const getHomeTypeBySPA = `
+query HomePageQuery {
+  getHomeTypeBy(input: {
+    entityType: "spa"
+  }) {
```
Comment (riginoommen) [inline packages/home-spa/src/service/service.js:5]: Home microservice is nearing deprication. It will be great if you can include the app service queries

Comment (hybridx) [reply packages/home-spa/src/service/service.js:5]: Is the app service available ?

**Review by riginoommen — CHANGES_REQUESTED** [review]

**Review by riginoommen — APPROVED** [review]

**Pull request merged**